paasta_tools/setup_kubernetes_cr.py

```diff
@@ -275,7 +281,9 @@ def format_custom_resource(
     url = get_dashboard_base_url(kind, cluster)
     if url:
         resource["metadata"]["annotations"][paasta_prefixed("dashboard_base_url")] = url
-
+    owner = get_cr_owner(kind.lower())
```

Comment:
nit: we're already lowercasing in `get_cr_owner()`, should we drop the lowercasing here?